cmake/InstallMSYS2.cmake

```diff
@@ -0,0 +1,117 @@
+    set(MINGW_LIB "${MINGW_PATH}/lib")
+
+    # /output/bin
+    file(GLOB MINGW_LIBS
+        ${MINGW_BIN}/libatk-1.0-[0-9]*.dll
+        ${MINGW_BIN}/libatkmm-1.6-[0-9]*.dll
+        ${MINGW_BIN}/libbz2-[0-9]*.dll
+        ${MINGW_BIN}/libcairo-[0-9]*.dll
+        ${MINGW_BIN}/libcairo-gobject-[0-9]*.dll
+        ${MINGW_BIN}/libcairo-script-interpreter-[0-9]*.dll
+        ${MINGW_BIN}/libcairomm-1.0-[0-9]*.dll
+        ${MINGW_BIN}/libdatrie-[0-9]*.dll
+        ${MINGW_BIN}/libdl.dll
+        ${MINGW_BIN}/libepoxy-[0-9]*.dll
+        ${MINGW_BIN}/libexpat-[0-9]*.dll
+        ${MINGW_BIN}/libfftw3-[0-9]*.dll
+        ${MINGW_BIN}/libfftw3f-[0-9]*.dll
+        ${MINGW_BIN}/libfribidi-[0-9]*.dll
+        ${MINGW_BIN}/libfontconfig-[0-9]*.dll
+        ${MINGW_BIN}/libfreetype-[0-9]*.dll
+        ${MINGW_BIN}/libgailutil-3-[0-9]*.dll
+        ${MINGW_BIN}/libgcc_s_seh-[0-9]*.dll
+        ${MINGW_BIN}/libgdk_pixbuf-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libgdk-3-[0-9]*.dll
+        ${MINGW_BIN}/libgdkmm-3.0-[0-9]*.dll
+        ${MINGW_BIN}/libgio-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libgiomm-2.4-[0-9]*.dll
+        ${MINGW_BIN}/libglib-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libglibmm_generate_extra_defs-2.4-[0-9]*.dll
+        ${MINGW_BIN}/libglibmm-2.4-[0-9]*.dll
+        ${MINGW_BIN}/libgmodule-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libgobject-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libgraphite[0-9]*.dll
+        ${MINGW_BIN}/libgthread-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libgtk-3-[0-9]*.dll
+        ${MINGW_BIN}/libgtkmm-3.0-[0-9]*.dll
+        ${MINGW_BIN}/libharfbuzz-[0-9]*.dll
+        ${MINGW_BIN}/libiconv-[0-9]*.dll
+        ${MINGW_BIN}/libintl-[0-9]*.dll
+        ${MINGW_BIN}/liblzma-[0-9]*.dll
+        ${MINGW_BIN}/libltdl-[0-9]*.dll
+        ${MINGW_BIN}/libMagick++-[A-Z0-9.-]*.dll
+        ${MINGW_BIN}/libMagickCore-[A-Z0-9.-]*.dll
+        ${MINGW_BIN}/libMagickWand-[A-Z0-9.-]*.dll
+        ${MINGW_BIN}/libpango-1.0-[0-9]*.dll
+        ${MINGW_BIN}/libpangocairo-1.0-[0-9]*.dll
+        ${MINGW_BIN}/libpangoft2-1.0-[0-9]*.dll
+        ${MINGW_BIN}/libpangomm-1.4-[0-9]*.dll
+        ${MINGW_BIN}/libpangowin32-1.0-[0-9]*.dll
+        ${MINGW_BIN}/libpcre-[0-9]*.dll
+        ${MINGW_BIN}/libpixman-1-[0-9]*.dll
+        ${MINGW_BIN}/libpng16-1[0-9]*.dll
+        ${MINGW_BIN}/libpng16-config
+        ${MINGW_BIN}/libquadmath-[0-9]*.dll
+        ${MINGW_BIN}/librsvg-2-[0-9]*.dll
+        ${MINGW_BIN}/libsigc-2.0-[0-9]*.dll
+        ${MINGW_BIN}/libssp-[0-9]*.dll
+        ${MINGW_BIN}/libstdc++-[0-9]*.dll
+        ${MINGW_BIN}/libthai-[0-9]*.dll
+        ${MINGW_BIN}/libtiff-[0-9]*.dll
+        ${MINGW_BIN}/libtiffxx-[0-9]*.dll
+        ${MINGW_BIN}/libwinpthread-[0-9]*.dll
+        ${MINGW_BIN}/libxml++-2.6-[0-9]*.dll
+        ${MINGW_BIN}/libxml2-[0-9]*.dll
+        ${MINGW_BIN}/zlib[0-9]*.dll
+    )
+    file(COPY ${MINGW_LIBS} DESTINATION ${SYNFIG_BUILD_ROOT}/bin)
+
+    set(MLT_DIRECTORY "${MINGW_PATH}/../opt/mlt-6.16.0")

```

Comment:
```
MLT_PATH="/opt/mlt-${VERSION_MLT}"
```
Since MLT is installed using an absolute path, there is no need to make it relative

Reply:
Going for an absolute path in CMake doesn't seem like a good option considering MSYS2 can be installed anywhere on the system and that we need path like `D:/msys64/opt` or whatever. That's why I kept it relative. 

Also, I'm unsure if we can find path on `msys` from CMake so that's why I kept it relative to `MINGW_PATH`.

Reply:
You can use `/opt/...` path, and translate it to windows path using `cygpath` tool